pkg/apis/application/v1alpha1/applicationset_types.go

```diff
@@ -804,7 +804,8 @@ type ApplicationSetStatus struct {
 	Conditions        []ApplicationSetCondition         `json:"conditions,omitempty" protobuf:"bytes,1,name=conditions"`
 	ApplicationStatus []ApplicationSetApplicationStatus `json:"applicationStatus,omitempty" protobuf:"bytes,2,name=applicationStatus"`
 	// Resources is a list of Applications resources managed by this application set.
-	Resources []ResourceStatus `json:"resources,omitempty" protobuf:"bytes,3,opt,name=resources"`
+	Resources      []ResourceStatus `json:"resources,omitempty" protobuf:"bytes,3,opt,name=resources"`
+	ResourcesCount int64            `json:"resourcesCount,omitempty" protobuf:"varint,4,opt,name=resourcesCount"`
```

Comment:
Probably wants a doc string so the future UI dev knows how to use this.

Are you hoping to cherry-pick the change back? I think for 3.2 it would be fine, but adding a CRD field further back feels maybe not great.

Reply:
ah, you are right. What do you think about adding `resourcesCount` in a separate PR? The UI functionality is not ready yet, so we are not loosing anything without `resourcesCount`. We can cherry-pick first PR into previous release and `resourcesCount` can go only to `master` branch.